apps/teams-test-app/e2e-test-data/hostEntity.tab.json

```
@@ -0,0 +1,79 @@
+{
+  "name": "Host entity Tabs",
+  "platforms": "Web",
+  "checkIsSupported": {
+    "capabilityName": "HostEntityTab",
+    "expectedOutput": "Host entity tab module is not supported"
+  },
```

```suggestion
  },
  "hostSdkVersion": {
    "web": ">4.0.1"
  },
```

`4.0.1` should be set to whatever the most recent release version of the hub-sdk is when this is checked in. Currently that's 4.0.1, but it may change between now and when you check in.